code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -67,6 +67,8 @@
 
 	var/mob/living/simple_animal/hostile/spawner/nest
 
+	var/sentience_type = SENTIENCE_ORGANIC // Sentience type, for slime potions

```

Comment:
is it really necessary to have that var? Right now it's only used for bots so we could just use has_unlimited silicon privilege instead. Unless we plan to restrict sentience potions for other animals in the future.